CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
 Changes
 =======
 
-# 5.15.0 / UNRELEASED
+# 5.16.0 / 07-26-2017
```

Comment:
Is this actually 5.16 or 5.17?

Reply:
They updated the changelog afterwards... doh. I don't think we should bother with updating this though.